### DISCUSSION TITLE
Webquerry POST doesn't work

### DISCUSSION THREAD

**Discussion by sisko215** [arendst/Tasmota#20410] [Q&A]
hello, 

i want to control a shelly device by sending a webquerry to the cloud. On a POST-tester it works, but on tasmota, the script do not work.

`WebQuery https://shelly-27-eu.shelly.cloud/device/relay/control/ POST {"id":"E****1", "auth_key":"N****2", "channel":"0", "turn":"on"}

Can anyone help me?`

**Comment by sfromis** [comment]
When making such a request to httpbin/anything, I can see that Tasmota successfully makes a post request to the test server running httpbin. Another question is if Shelly has special requirements for how such a request is made, like headers.

You could try making a request to `https://httpbin.org/anything` (or if you install it yourself) both ways, and see what's different.
